lib/features/modeling/cmd/LinkCallChoreoHandler.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * This handler is evoked when the user links a CallChoreo to a Choreography.
+ */
+export default function LinkCallChoreoHandler() {}
+
+/**
+ * Links the a choreo to a call choreo and un-links the old one if necessary.
+ * @param element {businessObject}
+ * @param newRef {businessObject} undefined if element should not be linked to anything
+ * @returns {*[]}
+ */
+LinkCallChoreoHandler.prototype.link = function(element, newRef) {
+  // we unlink the the element with the old choreo
+  if (element.calledChoreographyRef !== undefined) {
```

Comment:
In general (please check in the entire file), one should not compare to `undefined`.
Just use `if (element.calledChoreographyRef) { ... }`.